hq/app/aws/AwsAsyncHandler.scala

```diff
@@ -11,7 +11,7 @@ import scala.concurrent.{ExecutionContext, Future, Promise}
 class AwsAsyncPromiseHandler[R <: AmazonWebServiceRequest, T](promise: Promise[T], clientContext: AwsClient[_]) extends AsyncHandler[R, T] {
   def onError(e: Exception): Unit = {
     val context = Failure.contextString(clientContext)
-    Logger.warn(s"Failed to execute AWS SDK operation (${clientContext.client.getClass.getSimpleName}), $context", e)
+    Logger("aws").warn(s"Failed to execute AWS SDK operation (${clientContext.client.getClass.getSimpleName}), $context", e)
```

Comment:
Arguably this is an AWSPromiseHandler logger, rather than an aws one? I think using the name would help when someone's reading the logs later? This would be handled for us if we use the mixin trait (see my main comment as well)

Reply:
Have followed suggestion in main comment to use mixin trait instead 😸